DBTools/python/getSiblings.py

```diff
@@ -113,13 +131,193 @@ def setAPIs(self):
         elif args.prod == 'allUser':
             self.dbsapi_in = DbsApi (url = 'https://cmsweb.cern.ch/dbs/prod/phys03/DBSReader')
             self.dbsapi_out = DbsApi (url = 'https://cmsweb.cern.ch/dbs/prod/phys03/DBSReader')
+    '''
 
     #main function for getting siblings using either lumi matching or DBS
     def getSiblings(self):
         if args.prod == 'local' or args.prod == 'allLocal' or args.lumiMatching:
-            self.getLumiMatching(self.dictName, args.prod, args.inputJSON, args.siblingJSON)
+            #self.getLumiMatching(self.dictName, args.prod, args.inputJSON, args.siblingJSON)
+            print("Running local option...")
+            self.local = True
+            self.getFilesFromList(args.jobNumber, args.totalJobs)
+            self.findMatches()
         else:
-            self.getDBSSiblings(self.dictName)
+            self.getFilesFromList(args.jobNumber, args.totalJobs)
+            self.findMatches()
+
+    @staticmethod
+    def getDASInfo(dataset, jsonName=None):
+        cmd = 'dasgoclient -query="file,run,lumi,events dataset={0}" -json'.format(dataset)
```

Comment:
Are the json files large when also asking for events? If so, could it be an issue with storage space?

Reply:
largest I have seen was EGamma_2022F for miniAOD at 1.5Mb so shouldn't be an issue